### PR TITLE
Simplify Chinese translation of Scala Tour: self-types.md

### PR DESCRIPTION
**Pull request by realwunan**
Simplify Chinese translation of Scala Tour: Self-type

### CODE REVIEW

File: _zh-cn/tour/self-types.md [hunk 1]
```diff
@@ -13,3 +13,27 @@ language: zh-cn
...
+自类型是一种缩小 `this` 类型或 `this` 别名类型的方法。 语法看起来像普通函数语法，但是意义完全不一样。
+
+要在特质中使用自类型，写一个标识符，跟上要混入的另一个特质，以及 `=>`（例如 `someIdentifier: SomeOtherTrait =>`）。
+```tut
+trait User {
+  def username: String
+}
+
+trait Tweeter {
+  this: User =>  // 重新分配 this 的类型
```
Comment (liufengyun) [inline _zh-cn/tour/self-types.md]: 重新赋予

File: _zh-cn/tour/self-types.md [hunk 2]
```diff
@@ -13,3 +13,27 @@ language: zh-cn
 next-page: implicit-parameters
 previous-page: compound-types
 ---
+自类型用于声明一个特质必须混入其他特质，尽管该特质没有直接扩展其他特质。 这使得所依赖的成员可以在没有导入的情况下使用。
+
+自类型是一种缩小 `this` 类型或 `this` 别名类型的方法。 语法看起来像普通函数语法，但是意义完全不一样。
```
[the remark this answers is not in the document]
Comment (realwunan) [reply _zh-cn/tour/self-types.md]: > `缩小` -> `细化`/`强化`? 我感觉`强化`更好些，WDYT @realwunan

个人觉得细化更好些，英文原文用的是 narrow ，并且自类型确实限定了应该混入的类型

File: _zh-cn/tour/self-types.md (hunk 1 above)
Comment (realwunan) [reply _zh-cn/tour/self-types.md]: Thanks a lot.

**Comment by liufengyun** [comment]
Thanks a lot @realwunan  🎉

**Pull request merged**